quic/s2n-quic-core/src/datagram/default.rs

```diff
@@ -54,6 +62,9 @@ impl fmt::Display for BuilderError {
     }
 }
 
+/// Builder for the datagram endpoint
+///
+/// Use these methods to configure send/recv queue capacity
```

Comment:
```suggestion
/// Builder for the datagram endpoint
```

The builder might contain more later on. you can move the method comment to the fn themselves if you want.